src/Library/demos/Spinner/main.blp

```diff
@@ -0,0 +1,46 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Spinner";
+  description: _("Display loading state");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Box {
```

Comment:
this box is unecessary

try to keep the amount of UI needed to a minimum

set `halign: center;` on the `spinner` instead

Reply:
sorry - I meant 

set `halign: center;` on the `Button` instead

Reply:
okay